templates/commands/goldentest/verify.go

```diff
@@ -157,9 +160,24 @@ func (c *VerifyCommand) Run(ctx context.Context, args []string) error {
 				failureText := red(fmt.Sprintf("-- [%s] file content mismatch", goldenFile))
 				err := fmt.Errorf("%s:\n%s", failureText, dmp.DiffPrettyText(diffs))
 				tcErr = errors.Join(tcErr, err)
+				filesMismatch = true
 			}
 		}
 
+		if goldenContentFilesNotExist {
```

Comment:
Can we combine these into a single message? Like `Golden test contents didn't match actual output, you might need to ...`

Then we only need one bool and one `if` block.

Reply:
updated. I combined them but I think we still need two "bool"? How can it work with just one bool as file not exist and file not match are two cases.